src/components/CommitsPage/CommitGraphWrapper.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { ApolloError } from "@apollo/client";
+import styled from "@emotion/styled";
+import { Array } from "@ungap/global-this";
+import { Skeleton } from "antd";
+import { PageWrapper } from "components/styles";
+import { MainlineCommitsQuery } from "gql/generated/types";
+import { CommitGraph } from "./CommitGraph";
+
+export const CommitGraphWrapper: React.FC<{
+  versions: MainlineCommitsQuery["mainlineCommits"]["versions"];
+  error?: ApolloError;
+  graphType: "percentage" | "absolute";
+}> = ({ versions, error, graphType }) => {
+  let max = -1;
+  if (error) {
+    return <PageWrapper>ERROR</PageWrapper>;
+  }
+  if (!versions) {
```

Comment:
CommitGraphWrapper could also accept an `isLoading` prop that comes from the `useQuery` hook that also supplies the ApolloError. I think this would be more accurate view since it's possible for versions to be empty after network resources are done loading